app/views/field_slips/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>New field slip</h1>
```

Comment:
1. Consider a different title; I think many (most?) people will confuse a physical slip with a FieldSlip object, and wonder why they're making a "new" slip when they have one in front of them. 
But I don't have a great suggestion. Maybe "Link Observation to Field Slip".
2. Whatever the title, use a "translation". (Even if you don't anticipate use in forays outside the USA, some foray participants -- particularly at NAMA -- may not be native English speakers.)

Reply:
Interesting thought.  Thanks for catching the missing translation.  This naming came from the scaffold.  I generally need to fix up titles on the various pages.